.evergreen/config.in.yml

```diff
@@ -818,6 +818,40 @@ functions:
         args:
           - "${PROJECT_DIRECTORY}/.evergreen/run-snappy-version-test.sh"
 
+  "run lambda tests":
```

Comment:
Could consider a better name for these tasks (`run lambda tests with aws auth` as well)?  With the FAAS tests we may be adding soon, "run lambda tests" might be confusing because they don't actually run on lambda.

Something like `run lambda handler example tests` might be more clear but I'm open to suggestions on the name.

Reply:
I've updated both names to be more descriptive in that they are testing handler functions.